apps/builddao/widget/Compose.jsx

```diff
@@ -457,7 +457,7 @@ return (
               embedCss: props.customCSS || MarkdownEditor,
```

Comment:
@itexpert120 re: account autocomplete

buildhub.near/widget/components.MarkdownEditorIframe and the hander